resources/prompter_keep_focus.ahk

```diff
@@ -0,0 +1,11 @@
+#WinActivateForce
```

Comment:
Could you add a header to this file? Since AutoHotkey isn't used anywhere else in the project, it might be good to describe it a little better. Something like:
```
This is an AutoHotkey (https://www.autohotkey.com) script that is intended to run on a computer that displays the Sofie prompter page.
It automatically restores the focus to the Prompter-window if the focus is lost, so that the prompter controlling device (keyboard-equivalent / mouse-equivalent) continues to work.
```

Reply:
You're right. I've added the header to the script.